src/pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -57,9 +62,17 @@ def configure_logger(self, logger: Union[bool, Logger, Iterable[Logger]]) -> Non
             self.trainer.loggers = []
         elif logger is True:
             # default logger
-            self.trainer.loggers = [
-                TensorBoardLogger(save_dir=self.trainer.default_root_dir, version=SLURMEnvironment.job_id())
-            ]
+            if _TENSORBOARD_AVAILABLE or _TENSORBOARDX_AVAILABLE:
+                logger_ = TensorBoardLogger(save_dir=self.trainer.default_root_dir, version=SLURMEnvironment.job_id())
+            else:
+                warning_cache.warn(
+                    "Starting from release 1.9, `tensorboard` has been removed as a dependency of the `lightning`"
+                    " package, due to potential conflicts with other packages in the ML ecosystem. For this reason,"
+                    " `logger=True` will use `CSVLogger` as the default logger, unless the `tensorboard` package"
+                    " is found. Please `pip install lightning[extra]` to enable TensorBoard support by default"
```

Comment:
My suggestions might need some formatting.
```suggestion
                    " are found. Please `pip install lightning[extra]` or one of them to enable TensorBoard support by default"
```